src/main/java/frc/robot/subsystems/climberSubsystem/ClimberSubsystem.java

```diff
@@ -53,18 +53,15 @@ private void configMotor(CANSparkMax motor, boolean invert) {
     double armPosition;
 
     armPosition = motor.getAbsoluteEncoder().getPosition();
-
+    if(armPosition>0.5) {
+      armPosition = armPosition-1;
+    }
```

Comment:
I would change this to be:
`if(armPosition>0.5) armPosition -= 1`
its just a bit cleaner in my opinion